test/test_autograd.py

```diff
@@ -1542,6 +1542,14 @@ def func(root, b):
         go = Variable(torch.randn(a.size()), requires_grad=True)
         gradgradcheck(func, (a, b), (go,))
 
+    def test_inplace_view5(self):
```

Comment:
Can you please add docstrings to these tests? `test_inplace_view83` won't really tell you what's the essence of the problem and no one will remember that soon. Also, one line docstrings show up in test errors and verbose outputs, which is really nice